.github/workflows/docs-deploy.yml

```diff
@@ -0,0 +1,106 @@
+name: Docs Deploy
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
```

Comment:
Should this have a paths filter to the doc folder?

Reply:
yep will add